### ISSUE TITLE
Accessibility: Trouble using screen reader to interact with "write" link

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/wp-calypso#15887]
We received a report from a screen reader user who is unable to select the "write" link to the upper right of WordPress.com:

<img width="773" alt="screen-shot-2017-05-17-at-18-38-32 1" src="https://user-images.githubusercontent.com/2998162/27885274-4fe1f2e4-61cf-11e7-84a6-7d7bf4d97402.png">

They reported that nothing happens when clicking on that link. They have to manually right click the button, copy the link, then open it in a new tab.  

P2 for more discussion: p4vt7e-3f

The code that makes up that button is currently:

```
<div class="masterbar__publish>
    <a href="/post/siteaddress.com" title="Create a New Post" class="masterbar__item masterbar__item-new">
        <span class="masterbar__item-content"><!-- react-text: 25 -->Write<!-- /react-text --><!-- react-empty: 26 --></span>
    </a>
</div>
```

@davidakennedy offered the following insight:

> Right now, the element the user is interacting with is a `<a>` (link). They expect to point to a resource. It does and doesn’t here. It points to /post/sitename, but when you click, that’s intercepted by JS to open a sub-menu to search and select a site to post to. Some assistive technology won’t pick up on this without help. That explains why if the user copy and pastes the link, it does what they expect. I think they’re clicking and not getting that more elements are being expanded, etc.

> I’d change the `<a>` to a `<button>` and add some ARIA to it for greater context. Like aria-expanded attributes.

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

I was unable to reproduce this issue.

As there has been no recent activity on this issue, the issue is being closed with the label `Closed - Not Reproducible`.
If work is scoped and tabled for a milestone, please feel free to re-open this bug.
